### PR TITLE
Fix missing fstring on error message

### PR DESCRIPTION
**Pull request by marinegor**
Fixes #

Changes made in this Pull Request:
 - make string an fstring to ensure particular atom is printed correctly

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

## Developers certificate of origin
- [x] I certify that this contribution is covered by the LGPLv2.1+ license as defined in our [LICENSE](https://github.com/MDAnalysis/mdanalysis/blob/develop/LICENSE) and adheres to the [**Developer Certificate of Origin**](https://developercertificate.org/).

----
📚 Documentation preview 📚: https://mdanalysis--4638.org.readthedocs.build/en/4638/

### CODE REVIEW

**Review by hmacdope — APPROVED** [review]
LGTM

File: package/MDAnalysis/analysis/dssp/dssp.py [hunk 1]
```diff
@@ -319,7 +319,7 @@ def __init__(
                 ):
                     raise ValueError(
                         (
-                            "Residue {calpha.residue} contains wrong number of hydrogens: "
+                            f"Residue {calpha.residue} contains wrong number of hydrogens: "
```
Comment (IAlibay) [inline package/MDAnalysis/analysis/dssp/dssp.py:322]: Is this covered by a test? I.e. if it is, why did the error message matching not fail? If it isn't, then we should cover it by a test.

**Review by IAlibay — COMMENTED** [review]
Not blocking, but it seems like a test is either not working or doesn't exist.

File: package/MDAnalysis/analysis/dssp/dssp.py (hunk 1 above)
Comment (marinegor) [reply package/MDAnalysis/analysis/dssp/dssp.py:322]: @IAlibay fixed, thanks.

**Review by IAlibay — APPROVED** [review]
Amazing thanks!

**Pull request merged**